docs/design-v1.md

```diff
@@ -27,28 +29,34 @@ curl -X POST
 
 The response will include the following JSON fields:
 
-* `"status"` - "file_exists" | "no_file" | "error" - whether a file has been uploaded for this cache ID or not, or whether there was an error generating the ID.
+* `"status"` - "ok" | "error" - whether a file has been uploaded for this cache ID or not, or whether there was an error generating the ID.
 * `"id"`- the cache ID (plain string)
 * `"error"` - if "status" is "error", this will contain an error message
 
 An invalid service token will give a response status of 403, while other errors will give appropriate error status codes.
 
 **_Upload a file with a cache ID_**
 
-Post file data to the path `/cache/<cache_id>`. Use a content-type matching the file, such as `application/zip` or `application/octet-stream`.
+Post file data to the path `/cache/<cache_id>`. Use the `multipart/form-data` content type
 
 Headers:
 * `Authorization` - required - service token
-* `Content-Type` - required - content type of your file (such as `application/zip`)
-* `Content-Length` - required - length in bytes of the file
+* `Content-Type` - required - should be `multipart/form-data`
 
 A successful response will have JSON fields for:
 
-* `"status"` - "created" | "replaced" | "error". Whether a new file was created, a replacement was made, or there was an error
+* `"status"` - "ok" | "error". Whether the request was successful, or there was an error
 * `"error"` - if the status is "error", this will contain an error message
 
 An invalid service token will give a response status of 403, while other errors will give appropriate error status codes.
 
+```sh
+curl -X POST
+  -H "Content-Type: multipart/form-data"
+  -H "Authorization: <service_token>"
+  -F "file=@/home/user/my-file.txt"
+```
```

Comment:
Might be helpful to have examples with requests as well, but maybe that goes in README. Not sure what the line between the two is

Reply:
I agree, `requests` can have its own idiosyncracies. I'll add it in the readme